### DISCUSSION TITLE
open source contributions

### DISCUSSION THREAD

**Discussion by PaulineNemchak** [coder/coder#4392] [Announcements]
hello, @jsjoeio said in twitter that he could mentor with open source contributions. I have some experience with typescript (mainly with react.js) and don't know how to start contributing to outsource and a I'm a bit scared too because it seems that everyone is more experienced than me 🙈 could you please mentor me?

**Comment by jsjoeio** [comment]
hey @PaulineNemchak! 👋🏼 welcome to the Coder community! we'd love to help you! we can take it step-by-step and help you get there.

first things first - have you looked at the [`CONTRIBUTING.md`](https://github.com/coder/coder/blob/main/docs/CONTRIBUTING.md#contributing) yet?

see if you can get the repo downloaded locally and running. any time you get stuck, post to this thread and i'll help you! i check my notifications pretty frequently (M-F).
